docs/blitz/design/actions/actions_list.rst

```diff
@@ -50,7 +50,9 @@ Example can be seen below.
         timeout: 10
 
 The user can now change the result status to failed, passed, aborted, blocked, skipped, errored
-or passx when the step is passed. In the example below, result_status is changed to ``passx``.
+or passx when the step is passed. This feature is also supported by other actions like parse, api,
```

Comment:
now, I think better to have section for `result_status`.

Reply:
updated